tests/test_special.py

```diff
@@ -0,0 +1,86 @@
+import unittest
+
```

Comment:
please run these test on supported server version to make sure the queries work: 2005, 2008, 2012, 2016, 2017, and Azure SQLDB, Azure SQL DW

Reply:
tested

---
In reply to: [165537064](https://github.com/dbcli/mssql-cli/pull/155#discussion_r165537064) [](ancestors = 165537064)